apps/src/aichat/views/modelCustomization/RetrievalCustomization.tsx

```diff
@@ -61,30 +61,33 @@ const RetrievalCustomization: React.FunctionComponent = () => {
   return (
     <div className={modelCustomizationStyles.verticalFlexContainer}>
       <div className={modelCustomizationStyles.customizationContainer}>
-        <div className={modelCustomizationStyles.inputContainer}>
-          <FieldLabel
-            label="Retrieval"
-            id="retrieval-input"
-            tooltipText="Retrieval lets you add new information for a chatbot to reference. Type in each retrieval statement into the text box, then click “Add” for each one."
-          />
-          <textarea
-            id="retrieval-input"
-            onChange={event => setNewRetrievalContext(event.target.value)}
-            value={newRetrievalContext}
-            disabled={isReadOnly}
-          />
-        </div>
-        <div className={styles.addItemContainer}>
-          <Button
-            text="Add"
-            type="secondary"
-            color="gray"
-            size="s"
-            onClick={onAdd}
-            iconLeft={{iconName: 'plus'}}
-            disabled={!newRetrievalContext || isReadOnly}
-          />
-        </div>
+        {!isReadOnly && (
+          <>
+            <div className={modelCustomizationStyles.inputContainer}>
+              <FieldLabel
+                label="Retrieval"
+                id="retrieval-input"
+                tooltipText="Retrieval lets you add new information for a chatbot to reference. Type in each retrieval statement into the text box, then click “Add” for each one."
```

Comment:
Do you mind checking with product/curriculum to see if they would still want the Retrieval title and info icon at the top when read-only? Thanks!

Reply:
Thanks - LGTM!